Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -246,7 +243,7 @@ define([
         this._canRender = false;
         this._renderLoopRunning = false;
         this._showRenderLoopErrors = showRenderLoopErrors;
-        this._resolutionScale = 1.0;
+        this._resolutionScale = window.devicePixelRatio;
         this._forceResize = false;
         this._clock = defined(options.clock) ? options.clock : new Clock();
```

Comment:
In the `resolutionScale` getter below change the `@default` to `window.devicePixelRatio`

Reply:
This depends on the decision for https://github.com/AnalyticalGraphicsInc/cesium/pull/8057#discussion_r314772801